Assets/EditorCoroutines/Scripts/EditorCoroutines.cs

```diff
@@ -75,6 +75,20 @@ public bool IsDone(float deltaTime)
 			}
 		}
 
+		struct YieldCustomYieldInstruction : ICoroutineYield
+		{
+			public CustomYieldInstruction customYield;
+
+			public bool IsDone(float deltaTime)
+			{
+                /*
+                 * To keep coroutine suspended return true. To let coroutine proceed with execution return false.
+                 * From: https://docs.unity3d.com/ScriptReference/CustomYieldInstruction.html
+                 */

```

Comment:
Two small things here: 
1) The indentation is off, as it's spaces instead of tab. So it looks odd on GitHub. Better to fix that.
2) I think the comment here is not needed, the code is quite explicit in what it does. The comment only distracts for me.